content/source/docs/cloud/index.html.md

```diff
@@ -15,11 +15,11 @@ For expediency, this documentation refers to the application as Terraform Cloud.
 
 ## About Terraform Cloud and Terraform Enterprise
 
-[Terraform Cloud](https://www.hashicorp.com/products/terraform/) is an application that helps teams use Terraform together. It manages Terraform runs in a consistent and reliable environment, and includes easy access to shared state and secret data, access controls for approving changes to infrastructure, a private registry for sharing Terraform modules, detailed policy controls for governing the contents of Terraform configurations, and more.
+[Terraform Cloud](https://terraform.io/cloud) is an application that helps teams use Terraform together. It manages Terraform runs in a consistent and reliable environment, and includes easy access to shared state and secret data, access controls for approving changes to infrastructure, a private registry for sharing Terraform modules, detailed policy controls for governing the contents of Terraform configurations, and more.
 
-Terraform Cloud is available as a hosted service at [https://app.terraform.io](https://app.terraform.io). We offer free accounts for small teams, and paid plans with additional feature sets for medium-sized businesses.
+Terraform Cloud is available as a hosted service at [https://app.terraform.io](https://app.terraform.io). Small teams can sign up for free. As these teams scale, they can gain access to collaborative features through paid tiers. Finally, multiple teams within an organization can leverage [the Business tier](https://www.hashicorp.com/products/terraform/editions/cloud), which includes increased security, scalability, and visibility. 
```

Comment:
Since these are the docs it's better to be specific about the feature set. Show, don't tell :) 

```suggestion
Terraform Cloud is available as a hosted service at [https://app.terraform.io](https://app.terraform.io). Small teams can sign up for free to connect Terraform to version control, share variables, run Terraform in a stable remote environment, and securely store remote state. Paid tiers allow you to add more than five users, create teams with different levels of permissions, enforce policies before creating infrastructure, and collaborate more effectively. 

The [Business tier](https://www.hashicorp.com/products/terraform/editions/cloud) allows large organizations to scale to multiple concurrent runs, create infrastructure in private environments, manage user access with SSO, and automates self-service provisioning for infrastructure end users. 
```